modules/docs/Makefile.docs

```diff
@@ -1,4 +1,51 @@
+GOMPLATE_VERSION ?= v2.2.0
+GOMPLATE_REPO ?= hairyhenderson/gomplate
+GOMPLATE ?= gomplate
+
+TEMPLATES ?= $(BUILD_HARNESS_PATH)/templates/docs
+
 .PHONY : docs\:toc-update
 ## Update table of contents in README.md
 docs\:toc-update:
-	@doctoc --notitle --github README.md
\ No newline at end of file
+	@doctoc --notitle --github README.md
+
+## Install dependencies
+docs\:deps:
+	@make github:download-public-release \
+		REPO=$(GOMPLATE_REPO) \
+		FILE=$(GOMPLATE)_$(OS)-amd64 \
+		VERSION=$(GOMPLATE_VERSION) \
+		OUTPUT=/usr/local/bin/$(GOMPLATE) > /dev/null
+	@chmod +x /usr/local/bin/$(GOMPLATE)
+
+.PHONY : docs\:create
+## Create basic minimalistic .README.md template file
+docs\:create:
+	@cp $(BUILD_HARNESS_PATH)/templates/README.md ./.README.md
+
+.PHONY : docs\:build
+## Create README.md by building it from .README.md file
+docs\:build:
```

Comment:
I think this should be called `build-readme` since it's very specific to readme. Or `docs:` should be renamed to `readme:`

Reply:
Done